### PR TITLE
AO-871: Added IntegrationEmail entity with CRUD and db migration

### PR DESCRIPTION
**Pull request by sradziszewski**
JIRA task link: https://openlmis.atlassian.net/browse/AO-871

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -5,6 +5,8 @@ services:
     image: openlmis/referencedata
     env_file: .env
     depends_on: [db, log, redis]
+    ports:
+      - "5005:5005"
```
Comment (tsznaj) [inline docker-compose.yml:9]: That was added here accidentally, I think :)

File: package-lock.json
Comment (tsznaj) [inline package-lock.json:1]: That was added here accidentally, I think :)

File: src/main/java/org/openlmis/referencedata/repository/IntegrationEmailRepository.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      + "    ie.*\n"
+      + "FROM\n"
+      + "    referencedata.integration_emails ie\n"
+      + "WHERE\n"
+      + "    id NOT IN (\n"
+      + "        SELECT\n"
+      + "            id\n"
+      + "        FROM\n"
+      + "            referencedata.integration_emails ie\n"
+      + "            INNER JOIN referencedata.jv_global_id g "
```
Comment (tsznaj) [inline src/main/java/org/openlmis/referencedata/repository/IntegrationEmailRepository.java:41]: no newline character here - can it be like this?

File: docker-compose.yml (hunk 1 above)
Comment (sradziszewski) [reply docker-compose.yml]: I removed these lines.

File: src/main/java/org/openlmis/referencedata/repository/IntegrationEmailRepository.java (hunk 2 above)
Comment (sradziszewski) [reply src/main/java/org/openlmis/referencedata/repository/IntegrationEmailRepository.java:41]: I copied all this code from another class, and I see that it's the same in other classes, so it's not a problem. I'll leave it as is.

File: src/main/resources/db/migration/20240710090354134__add_integration_emails_table.sql
Comment (tsznaj) [inline src/main/resources/db/migration/20240710090354134__add_integration_emails_table.sql:1]: Do we need primary key here?

**Review by tsznaj — CHANGES_REQUESTED** [review]
Added one comment.

File: src/main/resources/db/migration/20240710090354134__add_integration_emails_table.sql
Comment (sradziszewski) [reply src/main/resources/db/migration/20240710090354134__add_integration_emails_table.sql:1]: I added primary key constraints.

**Pull request merged**